docs/sources/basics/exemplars.md

```diff
@@ -0,0 +1,22 @@
++++
+title = "Exemplars"
+description = "Exemplars"
+keywords = ["grafana", "concepts", "exemplars", "prometheus"]
+weight = 400
++++
+
+# Exemplars
```

Comment:
To match the rest of the basics, a lot of which will be something like, "What is logging? What are metrics?", Please change this to something that indicates this is a basic-level concept topic.

What are exemplars?
Intro to exemplars

Don't forget to update the file name to match.